src/DocBlock/Tags/InvalidTag.php

```diff
@@ -104,7 +104,9 @@ static function (&$value) : void {
         do {
             $trace = array_map(
                 static function (array $call) use ($flatten) : array {
-                    array_walk_recursive($call['args'], $flatten);
+                    $args = $call['args'] ?? [];
```

Comment:
This is not gonna work. you are changing the original behavior here. `$args` should be set to `$call['args'] to make it work as before. better would be to use `$call['args'] = $call['args'] ?? [];` Which will overwrite `args` when it was not set. 

It is hard to test this behavior. because we are highjacking the stack trace here. Which you don't want to do in a normal case. One of the issues we were facing before this addition was that libraries could not serialize this tag.